Packs/DomainTools_Iris/Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -441,38 +480,407 @@ def domain_profile_command():
+            domaintools_context = indicators.get('domaintools') if should_include_context(include_context) else None
+
+            command_results_list.extend(
+                (
+                    CommandResults(
+                        outputs_prefix='Domain',
+                        outputs_key_field='Name',
+                        indicator=domain_indicator,
+                        outputs=domaintools_context,
+                        readable_output=human_readable_output,
+                        raw_response=result,
+                        ignore_auto_extract=True,
+                    ),
+                    CommandResults(
+                        outputs_prefix='DomainTools',
+                        outputs_key_field='Name',
+                        readable_output='see output for domain indicator',
+                        outputs=domaintools_context,
+                        ignore_auto_extract=True,
+                    ),
+                )
+            )
+    return_results(command_results_list)
+
+
+def domain_enrich_command():
+    """
+    Command to do a total profile of a domain using iris-enrich API endpoint.
+    e.g. !domaintoolsiris-enrich domain=domaintools.com
+    """
+    domain = demisto.args()['domain']
+    domain_list = domain.split(",")
+    domain_chunks = chunks(domain_list, 100)
+    include_context = demisto.args().get('include_context', False)
+    command_results_list: List[CommandResults] = []
+
+    for chunk in domain_chunks:
+        response = domain_enrich(','.join(chunk))
+        missing_domains = response.get("missing_domains")
+        for result in response.get('results', []):
+            human_readable_output, indicators = format_enrich_output(result)
+
+            if len(missing_domains) > 0:
+                human_readable_output += f"Missing Domains: {','.join(missing_domains)}"
+
+            domain_indicator = indicators.get('domain') if should_include_context(include_context) else None
+            domaintools_context = indicators.get('domaintools') if should_include_context(include_context) else None
+
+            command_results_list.extend(
+                (
+                    CommandResults(
+                        outputs_prefix='Domain',
+                        outputs_key_field='Name',
+                        indicator=domain_indicator,
+                        readable_output=human_readable_output,
+                        raw_response=result,
+                        ignore_auto_extract=True,
+                    ),
+                    CommandResults(
+                        outputs_prefix='DomainTools',
+                        outputs_key_field='Name',
+                        outputs=domaintools_context,
+                        readable_output='see output for domain indicator',
+                        ignore_auto_extract=True,
+                    ),
+                )
+            )
+
+
+    return_results(command_results_list)
```

Comment:
updated! I combined the indicator and domaintools context into a single `CommandResult`, but kept the `command_results_list` list because we support multiple domains here. Is this okay?